surveys/2024-annual-survey/questions.md

```diff
@@ -290,6 +290,23 @@ Type: select all that apply (optional)
 > in a separate question, though this is likely easier to find out through download
 > numbers.
 
+### What is your opinion on how fast Rust evolves?
+
+We want to know if you are able to keep up with the speed of development of the Rust language.
```

Comment:
Also, this wording puts implicitedly the blame on the user (_"are you good enough for Rust?"_). I would invert the polarity with something like "we want to know if the Rust PL development is moving too fast"

Reply:
Good point!